### PR TITLE
feat(youtube): move video settings to `Video` settings category

### PR DESCRIPTION
**Pull request by johnconner122**

### CODE REVIEW

**Comment by oSumAtrIX** [comment]
On another note, do you consider these patches miscellaneous or interaction?

**Comment by johnconner122** [comment]
> On another note, do you consider these patches miscellaneous or interaction?

We are not constantly interacting, we just set the quality or speed, and forget about it.

**Comment by oSumAtrIX** [comment]
@LisoUseInAIKyrios The four menus are currently meant to be pages to hold a list of patch menus/settings. If we add another menu, just for a single patch, then this would break that convention, because now a fifth menu would soley be dedicated for the speed/quality patch. Keep in mind, the Settings patch is used the following way: "SettingsPatch.MISC.add(setting)". If you now add a quality/speed menu, now other patches would for some reason see: "SettingsPatch.QUALITY.add(setting)". Therefore it should be added to an existing menu and not in a dedicated one.

**Comment by oSumAtrIX** [comment]
If you introduce this menu, other video related settings should be moved into it.

**Comment by johnconner122** [comment]
I guess we are fine with current PR. 🤷

**Comment by johnconner122** [comment]
Looks good.

**Comment by oSumAtrIX** [comment]
Things like video timestamp don't belong in video playback menu.

**Comment by oSumAtrIX** [comment]
The same applies to them, the menu is labeled as "Settings related to [...] Playback" which is where the conflict happens

**Comment by oSumAtrIX** [comment]
The "root" menu is meant as categories. Is "speed" and "quality" a category? Are there patches that can be put inside the category in the future? If these menus are just made to host the current speed and quality settings, it loses it's meaning, which is why it should be inside the misc category instead.

File: src/main/kotlin/app/revanced/patches/youtube/misc/settings/bytecode/patch/SettingsPatch.kt [hunk 1]
```diff
@@ -137,6 +137,7 @@ class SettingsPatch : BytecodePatch(
         val ADS = Screen("ads", "Ads", "Ad related settings")
         val INTERACTIONS = Screen("interactions", "Interaction", "Settings related to interactions")
         val LAYOUT = Screen("layout", "Layout", "Settings related to the layout")
+        val VIDEO = Screen("video", "Video", "Settings related to video playback")
```
Comment (oSumAtrIX) [inline src/main/kotlin/app/revanced/patches/youtube/misc/settings/bytecode/patch/SettingsPatch.kt:140]:
```suggestion
        val VIDEO = Screen("video", "Video", "Settings related to the video player")
```

**Review by oSumAtrIX — CHANGES_REQUESTED** [review]
A new package should be created. Currently, the category has been changed from `misc` to `video`, but no package `video` has been introduced.

**Review by oSumAtrIX — APPROVED** [review]

File: src/main/kotlin/app/revanced/patches/youtube/misc/video/speed/VideoSpeeds.kt [hunk 2]
```diff
@@ -1,24 +0,0 @@
-package app.revanced.patches.youtube.misc.video.speed
```
Comment (johnconner122) [inline src/main/kotlin/app/revanced/patches/youtube/misc/video/speed/VideoSpeeds.kt:1]: @oSumAtrIX This file is empty due to some reasons!

Comment (oSumAtrIX) [reply src/main/kotlin/app/revanced/patches/youtube/misc/video/speed/VideoSpeeds.kt:1]: Please open a follow up PR, seems like this was caused by the merge conflict resolution.

**Pull request merged**